### PR TITLE
Gift Entry: use custom lookup component in template builder and batch wizard

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes
- Use `geFormFieldLookup` in Template Builder and Batch Wizard
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by psadie-sfdo** [comment]
**lurch: attach W-038443

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HqKKQA0" target="_blank">W-038443</a>

File: src/lwc/utilInput/utilInput.js [hunk 1]
```diff
@@ -16,11 +16,15 @@ const DATETIME = 'datetime-local';
 const YES = 'Yes';
 const TEXT = 'text';
 const TRUE = 'true';
+const RICH_TEXT_FORMATS = [
```
Comment (ptewson-sfdo) [inline src/lwc/utilInput/utilInput.js]: What is this being used for? It doesn't seem like this + richTextFormats is actually being used, unless I'm missing something?

Comment (psadie-sfdo) [reply src/lwc/utilInput/utilInput.js]: Ah, yes this is the whitelist of formats being set in the `lightning-input-rich-text` component's `formats` property. That property controls some of the buttons and what formatted content can be pasted into that rich input.

I straight up lifted this list from the one being used in the renderer so we have alignment across Gift Entry with regards to rich text inputs.

**Pull request merged**